src/Db/Entity/Toggle.php

```diff
@@ -0,0 +1,111 @@
+<?php
+namespace Clearbooks\Labs\Db\Entity;
+
+class Toggle extends CamelCaseMapperEntity
```

Comment:
Hm yeah I could, but I don't really think that there will be multiple implementations of this ever. What benefits would it have? My tests don't require it either.